starknet_py/hash/hash_method.py

```diff
@@ -14,7 +14,14 @@ class HashMethod(Enum):
     PEDERSEN = "pedersen"
     POSEIDON = "poseidon"
 
-    def hash(self, values: List[int]):
+    def hash(self, left: int, right: int):
```

Comment:
Not sure why we need this method. Why can't we simply pass a list with two elements instead?

Reply:
Imho it's better to keep `hash()` and `hash_many()` as separate methods. Generally hash methods for two elements produce different output than for many elements. For instance, such code
```python
print(poseidon_hash(1, 2))
print(poseidon_hash_many([1, 2]))
```
gives following output
```python
2636648219362971850283425434366427370362725365790740855428580782178634926362
1557996165160500454210437319447297236715335099509187222888255133199463084263
```